runtime/devnet/src/config/assets.rs

```diff
@@ -29,6 +29,8 @@ parameter_types! {
 
 parameter_types! {
 	pub NftsPalletFeatures: PalletFeatures = PalletFeatures::all_enabled();
+	// Key = 68 bytes (4+16+32+16), Value = 44 bytes (4+32+8)
+	pub const NftsBalanceDeposit: Balance = deposit(1, 112);
```

Comment:
The max size in the weights file shows account balance value as 120?

Reply:
The tests was incorrect because it used the configuration of mock runtime in the pallet not devnet / testnet / mainnet. Resolved by moving the test to its specific runtimes and updated the comments.